package.json

```diff
@@ -52,19 +51,19 @@
     "validator": "13.7.0"
   },
   "scripts": {
-    "pre-commit": "yarn prettier:check && yarn lint",
+    "pre-commit": "npx tsc --noEmit && yarn prettier:check && yarn lint",
     "hooks:uninstall": "husky uninstall",
     "hooks:install": "husky install",
-    "start": "env-cmd -f ./.env.local react-scripts start",
-    "start:test": "env-cmd -f ./.env.test react-scripts start",
-    "start:prod": "env-cmd -f ./.env.production react-scripts start",
-    "start:ci": "react-scripts -r @cypress/instrument-cra start",
-    "build": "react-scripts build",
-    "build:local": "env-cmd -f ./.env.local react-scripts build",
-    "dist": "env-cmd -f ./.env.production react-scripts build",
-    "dist:dev": "env-cmd -f ./.env.development react-scripts build",
-    "test": "env-cmd -f ./.env.test react-scripts test",
-    "test:once": "env-cmd -f ./.env.test react-scripts test --watchAll=false",
+    "start": "env-cmd -f ./.env.local react-app-rewired start",
+    "start:test": "env-cmd -f ./.env.test react-app-rewired start",
+    "start:prod": "env-cmd -f ./.env.production react-app-rewired start",
+    "start:ci": "react-app-rewired -r @cypress/instrument-cra start",
+    "build": "react-app-rewired build",
+    "build:local": "env-cmd -f ./.env.local react-app-rewired build",
+    "dist": "env-cmd -f ./.env.production react-app-rewired build",
+    "dist:dev": "env-cmd -f ./.env.development react-app-rewired build",
+    "test": "env-cmd -f ./.env.test react-app-rewired test",
+    "test:once": "env-cmd -f ./.env.test react-app-rewired test --watchAll=false",
```

Comment:
Do we use these ? I think we only use cypress tests and not react-testing library, but i may be wrong.